python/ambassador/ir/irambassador.py

```diff
@@ -252,31 +252,20 @@ def finalize(self, ir: 'IR', aconf: Config) -> bool:
             ir.save_filter(self.grpc_web)
 
         if amod and ('grpc_stats' in amod):
-            grpc_stats = amod.grpc_stats
-
+            grpc_stats = { 'all_methods': False} if amod.grpc_stats is None else amod.grpc_stats
```

Comment:
Hmm, I'd thought about the `grpc_stats: {}` case, but not the `grpc_stats: null` case.  I'm not thinking that they should be the same.  Notably, when we structurally type the `Module` resource, the apiserver will consider `null` and absence to be the same thing (not set). Whereas `{}` is set-but-empty .

Perhaps change the condition to ` if amod and (grpc_stats := amod.get('grpc_stats')) is not None:`